.editorconfig

```diff
@@ -18,7 +18,7 @@ indent_size = 4
 tab_width = 4
 
 # New line preferences
-end_of_line = crlf
+end_of_line = lf
```

Comment:
Switched this to ensure consistent formatting between Windows/non-Windows OS

Reply:
If we didn't want to update this, you can update your git config and rider to ensure it always commits properly. No preference personally. LF would make things easier for new contributors.